client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -124,6 +135,9 @@ const LoggedInForm = React.createClass( {
 		const { autoAuthorize, queryObject } = this.props.jetpackConnectAuthorize;
 		debug( 'Checking for auto-auth on mount', autoAuthorize );
 		if ( autoAuthorize || this.props.calypsoStartedConnection ) {
+			this.props.recordTracksEvent( 'jpc_auth_view', {
+				user: this.props.userId
```

Comment:
`userid` is already recorded automatically if the user it logged in (as a global/super property). I don't think we need to duplicate it here.